Makefile.am

```diff
@@ -93,14 +93,8 @@ SRCDIR   = ${shell cd $(top_srcdir) ; pwd | sed -e 's,^[^:\\/]:[\\/],/,'}
 RPMDIR   = $(BUILDDIR)/rpmbuild
 
 .PHONY: rpm
-rpm: control/ipmitool.spec dist
-	mkdir -p $(RPMDIR)/{BUILD,RPMS,SRPMS,SOURCES,SPECS,tmp}
-	cp control/ipmitool.spec $(RPMDIR)/SPECS
-	cp $(distdir).tar.gz $(RPMDIR)/SOURCES
-	CONFIGDIR=$(BUILDDIR)/control \
-	  @RPMBUILD@ -ba --define "_topdir $(RPMDIR)" \
-	  --rcfile $(SRCDIR)/control/rpmrc \
-	  $(RPMDIR)/SPECS/ipmitool.spec
+rpm: dist
+	@RPMBUILD@ -ta $(distdir).tar.gz
```

Comment:
What's the urge to build in ~/rpmbuild instead of where it was building? Was there any problem with that? If so, it needs an explanation in the commit log.

Reply:
No urge, but now once you've run "make dist" to create a source archive you can give it to somebody else and have it run simply "rpmbuild -tb ipmitool-*.tar.gz" to build an RPM. This is also the standard location to build an RPM as a regular user. And this also makes the Makefile much simple (only 1 line instead of 8 for the "rpm" target)
Sorry, I've forgotten to explain that in the commit message.

Reply:
I see. Please do explain that in the commit message.

Reply:
Good news : I've been able to simplify things and keep the build directory unchanged : under ./rpmbuild for "make rpm", and ~/rpmbuild/ for "rpmbuild -tb ipmitool-*.tar.gz"